sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/bridge/ValidatingStargateBridge.java

```diff
@@ -49,6 +49,12 @@ public Uni<QueryOuterClass.Response> executeQuery(QueryOuterClass.Query query) {
     return expectation.execute(null, query.hasParameters() && query.getParameters().getEnriched());
   }
 
+  @Override
+  public Uni<Schema.QueryWithSchemaResponse> executeQueryWithSchema(
+      Schema.QueryWithSchema request) {
+    throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
why just not return same as `executeQuery`.. I don't think we will be using schema failure in validating tests..

Reply:
Fixed in latest version.